ggshield/core/constants.py

```diff
@@ -18,3 +19,8 @@ def _get_max_workers() -> int:
 DEFAULT_HMSL_URL = "https://api.hasmysecretleaked.com"
 AUTH_CONFIG_FILENAME = "auth_config.yaml"
 ON_PREMISE_API_URL_PATH_PREFIX = "/exposed"
+
+
+class IncidentStatus(Enum):
```

Comment:
```suggestion
class IncidentStatus(str, Enum):
```

You need to inherit from string in order to make the string formatting work